cli/tests/pcluster/schemas/test_cluster_schema.py

```diff
@@ -590,6 +590,45 @@ def test_slurm_flexible_queue(mocker, config_dict, failure_message):
             },
             "Multiple .*Settings sections cannot be specified in the SharedStorage items",
         ),
+        (
+            {
+                "StorageType": "FsxFileCache",
+                "Name": "name",
+                "MountDir": "mount/tmp",
+                "FsxFileCacheSettings": {"FileCacheId": "fc-123456789012345678"},
+                "EfsSettings": {"Encrypted": True},
+            },
+            "Multiple .*Settings sections cannot be specified in the SharedStorage items",
+        ),
+        ({"StorageType": "FsxFileCache", "MountDir": "mount/tmp"}, "Missing data for required field."),
```

Comment:
*Minor/Code Style*
It's a good practice to document each test case with a description because:
1. it's easier for the reader to understand the goal of the test case when reading the code.
2. it's easier for a developer to understand what test case is failing when running the test locally.
You can see examples [here](https://github.com/aws/aws-parallelcluster/blob/develop/cli/tests/pcluster/templates/test_directory_service.py#L26-L36).

An argument may be that: we are not using this habit constantly in our codebase and that's true.
My suggestion is to use it at least on new test cases.
If using in your test cases onyl will make the test fail, then we will come back with a refactoring activity on a separate PR, otherwise I suggest to do that.

Reply:
I think adding comments for the newly added test case should suffice, if you agree on that I will go ahead with the change.

Reply:
Adding comments does not provide the same value because that approach does not provide advantage (2).
I prefer the pytest native approach, but I'm ok either ways.